### PR TITLE
added cluster zones doc

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Adds documentation for cluster zones

# Impacts
Documentation

# Delete branch after merge
YES

### CODE REVIEW

**Comment by extrafu** [comment]
Excellent documentation 👍

**Comment by julsemaan** [comment]
@fdurand 

Want to take a look (including the schema) to make sure I'm not saying anything stupid there.

Also, I'm testing the doc this morning

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 1]
```diff
@@ -645,6 +645,289 @@ As the daily automatic backups, you will find the files in:
...
+ * RFC3576 servers (CoA and RADIUS disconnect) should be declared on your network equipment (if supported) for both virtual IPs (192.168.1.10 and 192.168.2.10 in this example)
+
+RADIUS server with captive-portal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+WARNING: As shortly explained above, deploying a captive-portal across a layer 3 network using the cluster zones is complex and requires in-depth knowledge of networking and how PacketFence works.
+
+In order to configure a RADIUS server with a captive-portal on a layer 3 cluster, you will need at least 3 servers (5 are used in this example) with 2 interfaces (one for management and one for registration)
+
+NOTE: Isolation is omitted in this example for brievety and should be configured the same way as registration if needed
```
Comment (extrafu) [inline docs/PacketFence_Clustering_Guide.asciidoc]: for brevity

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 2]
```diff
@@ -645,6 +645,289 @@ As the daily automatic backups, you will find the files in:
...
+ * This example will use 3 servers in a network (called DC1), and 2 in another network (called DC2).
+ * Each group of server (in the same L2 network) will have a virtual IP address and will perform load-balancing (RADIUS, HTTP) to members in the same L2 zone (i.e. same network).
+ * All the servers will use MariaDB Galera cluster and will be part of the same database cluster meaning all servers will have the same data.
+ * In the event of the loss of DC1 or a network split between DC1 and DC2, the databases on DC2 will go in read-only and will exhibit the behavior described in "Quorum behavior".
+ * All the servers will share the same configuration and same cluster.conf. The data in cluster.conf will serve as an overlay to the data in pf.conf and networks.conf to perform changes specific to each layer 3 zone.
+
+The schema below presents the routing that needs to be setup in your network in order to deploy this example:
+
+image::cluster-zones-registration.png[scaledwidth="100%",alt="Cluster zones with registration network"]
+
```
Comment (fdurand) [inline docs/PacketFence_Clustering_Guide.asciidoc:758]: The schema is a little bit confusing for me and maybe we will need to add another network for routing.
Per example use 192.168.33.0/30 (to reach DC1) and 192.168.33.4/30 (to reach DC2).

**Comment by julsemaan** [comment]
Fixed typo and adjusted schema, @fdurand please check the schema again and merge if we're all good

**Pull request merged**